test/powershell/Modules/Microsoft.PowerShell.Utility/Remove-Alias.Tests.ps1

```diff
@@ -0,0 +1,45 @@
+Describe "Remove-Alias" -Tags "CI" {
+    It "Remove-Alias should remove a non-readonly alias"{
+        {
+            Set-Alias -Name "foo" -Value "bar"
+            Remove-Alias -Name "foo"
+            Get-Alias -Name "foo" -ErrorAction Stop
```

Comment:
I think we should keep a consistency and add `-ErrorAction Stop` to previous two lines - that will exclude _false positives_.
Please add this in tests below.